packages/fluentui/react-context-selector/src/useContextSelector.ts

```diff
@@ -18,39 +12,70 @@ export const useContextSelector = <Value, SelectedValue>(
   context: Context<Value>,
   selector: ContextSelector<Value, SelectedValue>,
 ): SelectedValue => {
-  const { subscribe, value } = React.useContext((context as unknown) as Context<ContextValue<Value>>);
-  const [, forceUpdate] = React.useReducer((c: number) => c + 1, 0) as [never, () => void];
+  const contextValue = React.useContext((context as unknown) as Context<ContextValue<Value>>);
```

Comment:
what kind of error do you get without this cast ?

Reply:
Conversion of type 'Context<Value>' to type 'Context<ContextValue<Value>>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.

Reply:
@layershifter 

If we do this change:
```typescript
export const useContextSelector = <Value, SelectedValue>(
  context: Context<ContextValue<Value>>,
```

We don't need this cast, WDYT?